dev/TabView/TabView.cpp

```diff
@@ -504,15 +597,115 @@ winrt::DependencyObject TabView::ContainerFromIndex(int index)
     return nullptr;
 }
 
-void TabView::OnCtrlF4Invoked(const winrt::KeyboardAccelerator& sender, const winrt::KeyboardAcceleratorInvokedEventArgs& args)
+int TabView::GetItemCount()
+{
+    if (auto itemssource = ItemsSource())
+    {
+        return winrt::make<InspectingDataSource>(ItemsSource()).Count();
```

Comment:
I use this to get the Count of the ItemsSource (rather than having to QI for the various possible interfaces and run through the iterator). Since we are planning on changing how TabView handles its Items/ItemsSource, this might be able to go away soon (e.g. if TabView is managing it's own Items collection).